app/javascript/src/controllers/media_tag_controller.js

```diff
@@ -25,8 +25,9 @@ export default class extends Controller {
   afterValidate(result, completeCallback) {
     if (result.authResponse.status === 'success') {
       this.initializeVideoJSPlayer()
-      const event = new CustomEvent('auth-success')
-      window.dispatchEvent(event)
+      if (result.authResponse.access_restrictions.stanford_restricted === true)
```

Comment:
this is what takes advantage of https://github.com/sul-dlss/stacks/pull/1049 to display the message that the user is logged in on a fresh page load, where they visit something after having already logged in on another page.